### PR TITLE
known_issues: Add some more known issues

### PR DESCRIPTION
**Pull request by rugeGerritsen**
* DRGN-15251: Very rare assertion fault when peripheral on Coded PHY 
* DRGN-15310: HCI Read RSSI fails

### CODE REVIEW

**Comment by thomasstenersen** [comment]
@ru-fu is there a way to indicate that a known issue has been fixed in current master? As it is today, you'd have to manually cross-check the release notes of your component from the last tag?

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -829,6 +829,23 @@ KRKNWK-8133: CSMA-CA issues
...
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+
+DRGN-15251: Very rare assertion fault when peripheral on Coded PHY
+  The controller could assert when the following conditions are met:
+
+  * The device is connected as a peripheral.
+  * The connection PHY is changed to LE Coded PHY.
+  * The devices have performed a data length update, the supported values are above the minimum specification defined values.
+  * A packet is received with a CRC error
```
Comment (thomasstenersen) [inline doc/nrf/known_issues.rst:840]: If the other bullet points ends with a period, this one should too :)

**Review by thomasstenersen — CHANGES_REQUESTED** [review]
Minor nitpick

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -829,6 +829,23 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+
+DRGN-15251: Very rare assertion fault when peripheral on Coded PHY
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:834]:
```suggestion
DRGN-15251: Very rare assertion fault when connected as peripheral on Coded PHY
```

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (ru-fu) [inline doc/nrf/known_issues.rst:840]:
```suggestion
  The controller might assert when the following conditions are met:

  * The device is connected as a peripheral.
  * The connection PHY is set to LE Coded PHY.
  * The devices have performed a data length update, and the supported values are above the minimum specification defined values.
  * A packet is received with a CRC error.
```

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -829,6 +829,23 @@ KRKNWK-8133: CSMA-CA issues
...
+  * The connection PHY is changed to LE Coded PHY.
+  * The devices have performed a data length update, the supported values are above the minimum specification defined values.
+  * A packet is received with a CRC error
+
+  **Workaround:** Do not enable :option:`CONFIG_BT_DATA_LEN_UPDATE` for applications that require Coded PHY as a peripheral device.
+
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+
+DRGN-15310: HCI Read RSSI fails
+  The command "HCI Read RSSI" returns "Command Disallowed (0x0C)"
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:847]:
```suggestion
  The command "HCI Read RSSI" always returns "Command Disallowed (0x0C)".
```

**Review by ru-fu — CHANGES_REQUESTED** [review]

**Comment by ru-fu** [comment]
> 
> 
> @ru-fu is there a way to indicate that a known issue has been fixed in current master? As it is today, you'd have to manually cross-check the release notes of your component from the last tag?

No, we do not mark issues that have been fixed on master. There was a discussion about adding a "master" tag as well, but it was rejected.

**Review by thomasstenersen — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -829,6 +829,23 @@ KRKNWK-8133: CSMA-CA issues
...
+  * The connection PHY is set to LE Coded PHY.
+  * The devices have performed a data length update, and the supported values are above the minimum specification defined values.
+  * A packet is received with a CRC error.
+
+  **Workaround:** Do not enable :option:`CONFIG_BT_DATA_LEN_UPDATE` for applications that require Coded PHY as a peripheral device.
+
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+
+DRGN-15310: HCI Read RSSI fails
+  The command "HCI Read RSSI" always returns "Command Disallowed (0x0C)"
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:847]:
```suggestion
  The command "HCI Read RSSI" always returns "Command Disallowed (0x0C)".
```

Comment (rugeGerritsen) [reply doc/nrf/known_issues.rst]: Fixed

**Review by carlescufi — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
Rebased on master to retrigger CI

**Pull request merged**